### ISSUE TITLE
Plot time domain methane

### ISSUE THREAD

**Issue by laserlab** [ubsuny/CP1-24-HW5#21]
Work on a single data analysis Jupyter notebook using functions from preparation.py that contain for each task below only one plot:

cleaned up and raw data by using some combination of waveform modification (which should include padding, windowing, taking the FFT, removing noise, inverse FFT, and undoing the window+padding) of the methane data on available continents in the time domain (1 member due Friday 2pm)

**Comment by ojha-aditya** [comment]
So I use the data filed compiled by @avgagliardo in PR #36 for this right?

**Comment by laserlab** [comment]
Yes, which is ready to merge and probably gets merged before class

**Comment by ojha-aditya** [comment]
@avgagliardo could you let me know which one of the data files contains te complete data for all continents?

**Comment by ojha-aditya** [comment]
@avgagliardo if you could explain the files in PR #36, I am a little confused which one I need to use for the task assigned above.

**Comment by avgagliardo** [comment]
> @avgagliardo if you could explain the files in PR #36, I am a little confused which one I need to use for the task assigned above.

I would use `methane_data.json` since it was picked to match the sites used for the CO2 data collection scrape. 

There is another file called `full_methane_data.json` which includes extra metadata (if anyone needed it).
